docs/virtual_machine_template_guide.adoc

```diff
@@ -0,0 +1,222 @@
+**One** OfflineVirtualMachine and **0-N** PersistentVolumeClaims. +
+The following document describes the process of creating a Virtual machine template which will be managed by ManageIQ.
+
+== Writing Templates
+You can define new VirtualMachine templates to make it easy to recreate all relevant objects (OfflineVirtualMachine and PersistentVolumeClaim). The template will define the objects it creates along with some metadata to guide the creation of those objects.
+
+[source,yaml]
+----
+apiVersion: kubevirt.io/v1alpha1
+kind: Template
+metadata:
+  name: RHEL-7 ❶
+  annotations:
+  labels:
+    miq.github.io/kubevirt-os: rhel-7 ❷
+    miq.github.io/kubevirt-is-vm-template: True ❸
+objects:
+- apiVersion: kubevirt.io/v1alpha1
+  kind: OfflineVirtualMachine ❹
+  metadata:
+    name: ${NAME} ➎
+  spec:
+    template:
+      metadata:
+        labels:
+          my: label
+      spec:
+        domain: 
+          resources: ➏
+            requests:
+              memory: ${MEMORY} 
+          cpu: ❼
+            cores: ${CPU}
+          devices:
+            disks: ❽
+              name: disk0
+              volumeName: mypvc 
+      volumes: ➒
+       - name: mypvc
+         presistentVolumeClaim:
+           claimName: rhel-7-pvc-${NAME}
+
+- apiVersion: v1
+  kind: PersistentVolumeClaim ➓
+  metadata:
+    name: rhel-7-pvc-${NAME} ⓫
+    #FIXME: add an annotation to specify which volume this pvc should be based on
+  spec:
+    accessModes:
+    - ReadWriteOnce
+    resources:
+      requests:
+        storage: 1Gi 
+parameters: ...
+
+----
+ 
+❶ The unique name of the template. +
+
+❷ OS identifier of the template. +
+
+❸ Indicator if this is a kubevirt template. +
+
+❹ The OfflineVirtualMachine Object. This object must appear only once in the template. +
+
+➎ OfflineVirtualMachine’s name. +
+
+➏ OfflineVirtualMachine’s resources, currently includes under requirements only memory. +
+   Memory is the minimal amount the Virtual Machine requires. +
+   These attributes are represented as parameters that will either be filled out by the user Or receive default values. +
+   
+❼ Number of cpu cores for the Virtual Machine. +
+  This attribute is represented as parameter that will either be filled out by the user Or receive a default value. +
+  
+❽ Disk contains a "volumeName" attribute, which links it to a volume +
+
+➒ Specification of a disk's volume from the disks section. It contains persistent volume claims.  
+(more about link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/[Persistent Volume Claims] ) +
+        
+➓ The PersistenVolumeClaim Object that accompanies the OfflineVirtualMachine Object. +
+
+⓫ PersistenVolumeClaims should include: name, permissions and requested amount of storage. +
+
+
+
+=== Labels
+
+Labels will be presented on every OCP VirtualMachineTemplate in order for ManageIQ to recognize the KubeVirt template characteristics. + 
```

Comment:
VirtualmachineTemplate still exists in the text.

Reply:
@rmohr  done